### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by coreypnorris**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,5 @@
 source 'https://rubygems.org'
 
-ruby '2.0.0'
-
```
Comment (dustinbrownman) [inline Gemfile:4]: It's weird that this is still an unmerged change...

Comment (geneeves) [inline Gemfile:4]: What's going on up there, are you guys refactoring previous students' code
as a class exercise?

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -645,4 +645,34 @@ bootstrap theme
...
+  cursor: pointer; background: #311;
+}
+input.toggle ~ div {
+  height: 1px; margin: .2rem;
+  overflow: hidden;
+  transition: 1s all cubic-bezier(0.730, -0.485, 0.145, 1.620)
+}
+input.toggle:checked ~ div { height: 190px; }
+input.toggle:checked + label { background: #ef669e; }
+input.toggle { display: none; }
```
Comment (dustinbrownman) [inline app/assets/stylesheets/custom.css.scss:678]: Not sure why this isn't already part of master...

**Comment by dustinbrownman** [comment]
So this is doing a lot more than just fixing merge conflicts. Might be fine for this one time, but should really try and avoid this in the future.

File: Gemfile (hunk 1 above)
Comment (dustinbrownman) [inline Gemfile:4]: Hey Geneeves! No, they're actually continuing on Chicktech as one of the
non-profit projects. It's very enlightening watching people jump into your
old code. We were super clever in some ways, but definitely not as great in
others.

How're you doing??

**Pull request merged**